src/types/generated/operations.ts

```diff
@@ -1,5 +1,23 @@
 import * as Types from "./types";
 
+export type TokenActivitiesFieldsFragment = {
```

Comment:
this change seems to come up in some different PRs (mine, Greg's and now yours) - not sure why

Reply:
ohhh good catch! 

Interesting, not sure why it shows up, I only updated the query for Processor_status. Maybe the codegen method has been updated?